### DISCUSSION TITLE
MIHI2Q import VW_STL_DB.sqlite

### DISCUSSION THREAD

**Discussion by BooBoo124** [Mr-MIBonk/M.I.B._More-Incredible-Bash#62] [General]
Hi,
I've patched my firmware MIQ2-ER-AUG22-P4212. Ive dropped the above sqlite file I downloaded from MIB solution and dropped it the RSDB folder. When I load up M.I.B and try to update the file I'm told there is no sqlite in the folder? I even downloaded the multi RSDB v1.10.10 but keep getting metainfo3 signature is not correct. I know I have to change variant to FMQ-*-*-*-* metainfo2 and 3 but no joy with that either. I e even tried newer version too.
Am I doing something wrong or do I need to update my firmware?

**Comment by harman-f** [comment]
File name must be `VW_STL_DB.sqlite` in `/mod/RSDB/`

**Reply by BooBoo124** [reply]
It is as it's the file I downloaded from m.i.b solutions. And that is where I put it as the is a txt file in there to say place here.

**Reply by harman-f** [reply]
Can you share a screenshot of the GEM output when you try to run the script.

**Comment by BooBoo124** [comment]
Hi I re-down loaded the file and it went on ok this time. Thanks for your time and great work youve done.
